source/Calamari.Tests/Fixtures/Util/AesEncryptionFixture.cs

```diff
@@ -0,0 +1,23 @@
+using System;
+using System.Globalization;
+using System.IO;
+using System.Xml.Linq;
+using Calamari.Util;
+using NUnit.Framework;
+using Octostache;
+
+namespace Calamari.Tests.Fixtures.Deployment
+{
+    [TestFixture]
+    public class AesEncryptionFixture
+    {
+        [Test]
+        public void CanGetRandomStrings() 
+        {
+            var randomString1 = AesEncryption.RandomString(8);
+            var randomString2 = AesEncryption.RandomString(8);
+
+            Assert.That(randomString1, Is.Not.EqualTo(randomString2));
```

Comment:
@pawelpabich The change to `AesEncryption.RandomString()` now use a single static `Random` instance and both of these use `.Next()` to get the random number.  This should be random.  This SO post is the best source on this.  

https://stackoverflow.com/questions/767999/random-number-generator-only-generating-one-random-number